lib/repo_tools.pm

```diff
@@ -133,6 +154,30 @@ sub rmt_mirror_repo {
     assert_script_run 'rmt-cli repo list';
 }
 
+=head2 rmt_import_data
+    rmt_import_data($datafile);
+RMT server import data about available repositories and the mirrored packages 
+from disconnected RMT server, then verify imported repos on new RMT server.
+=cut
+sub rmt_import_data {
+    my ($datafile) = @_;
+    my $datapath = "/mnt/external/";
+    # Decompress the RMT data file to test path
+    assert_script_run("mkdir -m 777 $datapath");
+    assert_script_run("wget -q " . data_url("rmt/$datafile"));
+    assert_script_run("tar -xzvf $datafile -C $datapath");
+    assert_script_run("rm -rf $datafile");
+    # Import RMT data from test path to new RMT server
+    assert_script_run("rmt-cli import data $datapath",  600);
+    assert_script_run("rmt-cli import repos $datapath", 600);
+    # Show repo list on new RMT server for later debugging
+    assert_script_run("rmt-cli repos list");
+    assert_script_run("rmt-cli repos enable SLE-Module-Web-Scripting15-SP1-Pool");
```

Comment:
What's about using `VERSION` variable here not to rewrite code for SP2?

Reply:
Yes, thanks.

Reply:
I found the 'repos enable' is no using for my test, so I removed this line.